src/root.cls.php

```diff
@@ -19,7 +19,52 @@ abstract class Root {
 	private static $_network_options = array();
 
 	/**
+	 * Check if there is cache folder for that type
+	 *
+	 * @since  3.0
+	 */
+	public function has_cache_folder( $type ) {
+		$subsite_id = is_multisite() && ! is_network_admin() ? get_current_blog_id() : '';
+
+		if ( file_exists( LITESPEED_STATIC_DIR . '/' . $type . '/' . $subsite_id ) ) {
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * Delete file-based cache folder for that type
+	 *
+	 * @since  3.0
+	 */
+	public function rm_cache_folder( $type ) {
+		if ( ! $this->has_cache_folder( $type ) ) {
+			return;
+		}
+
+		$subsite_id = is_multisite() && ! is_network_admin() ? get_current_blog_id() : '';
+
+		File::rrmdir( LITESPEED_STATIC_DIR . '/' . $type . '/' . $subsite_id );
+
+		// Clear All summary data
+		$this->_summary = array();
+		self::save_summary();
+
+		if ( $type == 'ccss' || $type == 'ucss') {
+			Debug2::debug( '[CSS] Cleared ' . $type .  ' queue' );
+		}
+		elseif ( $type == 'avatar' ) {
```

Comment:
Isn't the debug log in this condition same as the `else{}` one?

Reply:
This is not a big deal but can be improved still. I will approve this PR. You can include this improvement in your next PR.